backend/app/controllers/projects.js

```diff
@@ -414,14 +414,14 @@ module.exports = {
                 );
 
                 var productId = _.first(_.map((yield thunkQuery(
-                    Product.select(Product.id).from(Product).where(Product.projectId.equals(req.params.id))
+                    Product.select(Product.id).from(Product).where(Product.projectId.equals(req.params.projectId))
                 )), 'id'));
 
                 if (productId) {
                     return yield thunkQuery(
-                        'UPDATE "Tasks" ' +
-                        'SET "Tasks"."isDeleted" = '+ Date.now() +
-                        'WHERE "Tasks"."productId" = ' + productId +
+                        'UPDATE "Tasks"' +
+                        ' SET "isDeleted" = (to_timestamp('+ Date.now() +
```

Comment:
@jmfadeley I am very late on this but what does this do?

Reply:
NVM- I just saw Chris' comment. I wonder why it worked with postman and crashed when we did it from the front end.